packages/yoroi-extension/app/components/transfer/BaseTransferPage.js

```diff
@@ -59,28 +55,35 @@ export default class BaseTransferPage extends Component<Props> {
         actions={actions}
         closeOnOverlayClick={false}
         onClose={onBack}
-        className={styles.dialog}
+        className={undefined}
```

Comment:
If undefined, just remove the entire prop.

Reply:
Fixed